### ISSUE TITLE
zip file for v1.807 seems to have ttf v1.806 bundled

### ISSUE THREAD

**Issue by krish-r** [MonoLisaFont/feedback#132]
Hello, 

* Font edition: Personal
* Font version: 1.807
* Font Type: ttf

The .zip file for **v1.807** seems to have TTF **v1.806** bundled instead (OTF folder has v1.807). 
Can you please take a look into this?

**Comment by aleksejsolovev** [comment]
1.807 zip file has fixed TTF issue #128 for me, and SHA1 sums of TTF files in 1.806 and 1.807 bundles differ. But version tags of TTF files in 1.807 bundle left unchanged:
```
$ fc-scan MonoLisa-1.806-plus/otf/MonoLisa-Bold.otf | grep version
	fontversion: 118358(i)(s)
$ fc-scan MonoLisa-1.806-plus/ttf/MonoLisa-Bold.ttf | grep version
	fontversion: 118358(i)(s)
$ fc-scan MonoLisa-1.807-plus/otf/MonoLisa-Bold.otf | grep version
	fontversion: 118423(i)(s)
$ fc-scan MonoLisa-1.807-plus/ttf/MonoLisa-Bold.ttf | grep version
	fontversion: 118358(i)(s)
$ sha1sum MonoLisa-1.806-plus/ttf/MonoLisa-Bold.ttf
ace99add0bda39e227cec958c8aea54929a6c2a1  MonoLisa-1.806-plus/ttf/MonoLisa-Bold.ttf
$ sha1sum MonoLisa-1.807-plus/ttf/MonoLisa-Bold.ttf
866714c9eafac57f9fd9d9785631d56586c5eae4  MonoLisa-1.807-plus/ttf/MonoLisa-Bold.ttf
```
For me 1.807 bundle works perfect, and IMHO versions are very minor issue.

**Comment by bebraw** [comment]
Yes, it's a version tag issue.

@MarcusSterz Can you please attach the correct version tag to the new ttf so I can fix this one?

**Comment by krish-r** [comment]
ah! okay, Thanks to both of you for checking. 
Since this is a minor observation, I'm happy to close this issue if you think this can correct be as a part of next release.

**Comment by bebraw** [comment]
@krish-r No worries. It's worth it to get it fixed and maybe I can add a check to the process to verify versions. I'll close the issue then.

**Comment by bebraw** [comment]
Fixed in 1.808.
